utils/src/bulk-insert-patients.ts

```diff
@@ -0,0 +1,142 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+
+import { MetriportMedicalApi, PatientCreate, USState } from "@metriport/api";
+import csv from "csv-parser";
+import fs from "fs";
+
+const apiKey = getEnvVarOrFail("API_KEY");
+const facilityId = getEnvVarOrFail("FACILITY_ID");
+const apiUrl = getEnvVarOrFail("API_URL");
+
+const metriportAPI = new MetriportMedicalApi(apiKey, {
+  baseAddress: apiUrl,
+});
+
+async function main() {
+  const results: PatientCreate[] = [];
+  const errors: Array<{ firstName: string; lastName: string; dob: string; message: string }> = [];
+
+  fs.createReadStream("../insert-patients.csv")
+    .pipe(csv())
+    .on("data", async data => {
+      const metriportPatient = mapCSVPatientToMetriportPatient(data);
+      results.push(metriportPatient);
+    })
+    .on("end", async () => {
+      for (const [i, patient] of results.entries()) {
+        try {
+          await sleep(3000);
```

Comment:
NABD, but its easier to see params at the top of the file as `const` or env vars

Reply:
I get what your saying but what exactly are you referring too?

Reply:
The sleep function?

Reply:
just the amount... could create a const at the top of the file with this amount (3000)